### ISSUE TITLE
Plugin Error Unexpected token (5:22896) in  ..\node_modules\@stencil\state-tunnel\dist\esm\es5\stencil-state-tunnel.core.js

### ISSUE THREAD

**Issue by MoAsmar** [ionic-team/stencil#1063]
**Stencil version:**
 @stencil/core@0.9.11

**I'm submitting a:**
[ x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**
npm run build failed, below is the error:

[ ERROR ]  Plugin Error
           Unexpected token (5:22896) in
           C:\Workarea\app\node_modules\@stencil\state-tunnel\dist\esm\es5\stencil-state-tunnel.core.js

**Expected behavior:**
Build

**package.json:**

```tsx
{
  "name": "myapp",
  "version": "0.0.123",
  "description": "component",
  "main": "dist/index.js",
  "types": "dist/types/components.d.ts",
  "collection": "dist/collection/collection-manifest.json",
  "files": [
    "dist/"
  ],
  "module": "dist/index.js",
  "scripts": {
    "build": "stencil build --prerender",
    "dev": "sd concurrent \"stencil build --dev --watch\" \"stencil-dev-server\" ",
    "serve": "stencil-dev-server",
    "start": "npm run dev",
    "test": "jest --no-cache",
    "test.watch": "jest --watch --no-cache"
  },
  "dependencies": {
    "@stencil/router": "^0.2.2",
    "@stencil/sass": "0.0.3",
    "@types/jquery": "^3.3.4"
  },
  "devDependencies": {
    "@stencil/core": "^0.9.11",
    "@stencil/dev-server": "latest",
    "@stencil/utils": "latest",
    "@types/jest": "^21.1.1",
    "jest": "^21.2.1"
  },
  "jest": {
    "transform": {
      "^.+\\.(ts|tsx)$": "<rootDir>/node_modules/@stencil/core/testing/jest.preprocessor.js"
    },
    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(tsx?|jsx?)$",
    "moduleFileExtensions": [
      "ts",
      "tsx",
      "js",
      "json",
      "jsx"
    ]
  }
}

```

**Other information:**

**Comment by MoAsmar** [comment]
it seems the issue was because of a missing folder './stencil' but I don't know why it got stuck here, i had to clone the repo in another folder and install deps to get it working.

**Comment by jthoms1** [comment]
Was the error thrown during prerender?

**Comment by ryanwatkins** [comment]
Saw a similar error, with one component that imported another.  Deleting the .stencil directories fixed the issue.

**Comment by bradoyler** [comment]
Having same issue on windows... no idea why.

**Comment by tomsaunders** [comment]
After a lot of debug logging the stencil build script to try to work out where this was coming from, I got past this problem by specifying **0.2.5** for the **@stencil/router** dependency.

**Comment by MoAsmar** [comment]
I will try both of these solutions next time it happens with me, thanks @tomsaunders & @ryanwatkins

**Comment by MoAsmar** [comment]
@jthoms1 it was thrown on 'npm start' 
![image](https://user-images.githubusercontent.com/8542126/45210869-68099900-b299-11e8-8b73-a9f7d08ff6eb.png)

**Comment by rjacobson16** [comment]
Sorry, which `.stencil` directories do you mean? Having the same issue

**Comment by ryanwatkins** [comment]
In my case its the .stencil cache directory created by the compiler in the root of the component folder.

**Comment by Gafilipoie** [comment]
I'm experiencing the same issue, my stancil core is 0.11.2

**Comment by jthoms1** [comment]
If you are using Router 0.2.6 please use update Stencil to 0.12.4.

**Comment by MoAsmar** [comment]
I already fixed it by updating stencil to 0.11.4 and router to 0.2.5, thanks @jthoms1

**Comment by MoAsmar** [comment]
a little advice if it still happens even if you update stencil to 0.11.4 and router to 0.2.5, try to run:
`npm cache clear --force`
